dtfs-central-api/api-tests/v1/utilisation-reports/utilisation-reports-get-reconciliation-summary.api-test.js

```diff
@@ -1,36 +0,0 @@
-const { MONGO_DB_COLLECTIONS } = require('@ukef/dtfs2-common');
-const wipeDB = require('../../wipeDB');
-const app = require('../../../src/createApp');
-const { MOCK_BANKS } = require('../../mocks/banks');
-const api = require('../../api')(app);
-
-describe('/v1/utilisation-reports/reconciliation-summary/:submissionMonth', () => {
-  beforeAll(async () => {
-    await wipeDB.wipe([MONGO_DB_COLLECTIONS.BANKS]);
-    await api.post(MOCK_BANKS.BARCLAYS).to('/v1/bank');
-  });
-
-  describe('GET /v1/utilisation-reports/reconciliation-summary/:submissionMonth', () => {
-    it('returns a 200 response when the submissionMonth is a valid ISO month', async () => {
-      // Arrange
-      const submissionMonth = '2023-11';
-
-      // Act
-      const { status } = await api.get(`/v1/utilisation-reports/reconciliation-summary/${submissionMonth}`);
-
-      // Assert
-      expect(status).toEqual(200);
-    });
-
-    it('returns a 400 response when the submissionMonth is not a valid ISO month', async () => {
```

Comment:
This test doesn't seem to have been carried over is that intentional?

Reply:
Because it returns a `400`, I didn't see much value in adding any checks for the response body. Do you think we should add a check for the error message?